### PR TITLE
(api-cat) API-556 increase code coverage for SwaggerJsonSpecConverter

### PR DESCRIPTION
**Pull request by gitpiphub**

### CODE REVIEW

File: applications/api-cat/src/test/java/no/acat/spec/converters/SwaggerJsonSpecConverterTest.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  @Before
+  public void setup() throws IOException {
+    MockitoAnnotations.initMocks(this);
+    spec = IOUtillity.getStringOutputFromFile("swagger-raw-enhet-api.json");
+  }
+
+  @Test
+  public void test_if_canConvert() {
+
+    boolean convert = SwaggerJsonSpecConverter.canConvert(spec);
```
Comment (markkopaas) [inline applications/api-cat/src/test/java/no/acat/spec/converters/SwaggerJsonSpecConverterTest.java]: Igjen, hver test skulle ha klar fire steg
1)setup
2)operation
3)check
4)teardown

Hvis mulig, unngå shared state mellom test.

I din test her skulle du ha to linjer:
spec=loadResource...
assertTrue(canConvert(...)

File: applications/api-cat/src/test/java/no/acat/utils/IOUtillity.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package no.acat.utils;
+
+import org.apache.commons.io.IOUtils;
+import org.springframework.core.io.ClassPathResource;
+import java.io.IOException;
+
+public class IOUtillity {
+
+  public static String getStringOutputFromFile(String classPathResourceFileName) throws IOException {
+    return IOUtils.toString(
```
[the remark this answers is not in the document]
Comment (gitpiphub) [reply applications/api-cat/src/test/java/no/acat/utils/IOUtillity.java]: 👍

File: applications/api-cat/src/test/java/no/acat/spec/converters/SwaggerJsonSpecConverterTest.java (hunk 1 above)
Comment (gitpiphub) [reply applications/api-cat/src/test/java/no/acat/spec/converters/SwaggerJsonSpecConverterTest.java]: ok

File: applications/api-cat/src/test/java/no/acat/utils/IOUtillity.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package no.acat.utils;
```
[the remark this answers is not in the document]
Comment (gitpiphub) [reply applications/api-cat/src/test/java/no/acat/utils/IOUtillity.java]: skal fikses i en ny PR

File: applications/api-cat/src/test/java/no/acat/utils/IOUtillity.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+import org.apache.commons.io.IOUtils;
+import org.springframework.core.io.ClassPathResource;
+import java.io.IOException;
+
+public class IOUtillity {
+
+  public static String getStringOutputFromFile(String classPathResourceFileName) throws IOException {
```
[the remark this answers is not in the document]
Comment (gitpiphub) [reply applications/api-cat/src/test/java/no/acat/utils/IOUtillity.java]: endres getStringOutputFromClassPathResource, skal fikses i en ny PR.

**Pull request merged**